### PR TITLE
Ability to display country in language switcher

### PR DESCRIPTION
**Pull request by jtraulle**
Add ability to display country for language codes like fr_FR or fr_BE for example.

### CODE REVIEW

File: Switcher/TargetInformationBuilder.php [hunk 1]
```diff
@@ -50,16 +50,18 @@ public function __construct(Request $request, RouterInterface $router, $allowedL
...
-     * Will return something like this (let's say current locale is fr :
+     * Will return something like this (let's say current locale is fr_FR :
      *
      * current_route: hello_route
-     * current_locale: fr
+     * current_locale: fr_FR
      * locales:
-     *   en:
-     *     link: http://app_dev.php/en/... or http://app_dev.php?_locale=en
-     *     locale: en
```
Comment (dbu) [inline Switcher/TargetInformationBuilder.php]: could i configure both a locale with region and without? would be nice to keep this example and just add en_US additionally, to not risk the assumption that you need the region even if you do not care about it.

**Comment by dbu** [comment]
looks sane to me.

**Comment by ikwattro** [comment]
@jtraulle Can you add some tests for this behavior. Thanks!

**Comment by jtraulle** [comment]
@ikwattro Sorry but I do not have time at the moment :wink: 
Feel free to close this PR if you want.

**Pull request closed without merging**